docs/en/reference/configuration.rst

```diff
@@ -71,6 +76,10 @@ Now, in the root of your project place a file named ``migrations.php``, ``migrat
 
             <name>My Project Migrations</name>
 
+            <connection>default</connection>
+            <em>default</em>
```

Comment:
Why is it not null in xml (or whatever the notation for `null` is)?

Reply:
that is the right question, in XML to have it `null` the tag should not be there... but for documentation proposes i had to put something.... and `default` is the default name for a connection or entity manager